game/fs.cc

```diff
@@ -77,6 +77,25 @@ namespace {
 		return ret;
 	}
 
+	nlohmann::json readJSON(fs::path const& filename) {
+		nlohmann::json json;
```

Comment:
`json` isn't assigned/initialized a value. Thus a calling `.empty()` on the caller side (without checking for NULL) can result in a error right?